datafusion/core/src/datasource/file_format/parquet.rs

```diff
@@ -220,7 +220,7 @@ impl FileFormat for ParquetFormat {
 }
 
 fn summarize_min_max(
-    max_values: &mut [Option<MaxAccumulator>],
+    max_values: &mut [Option<SlidingMaxAccumulator>],
```

Comment:
I don't understand this change

Reply:
> Other than the changes in `datafusion/core/src/datasource/file_format/parquet.rs` and `datafusion/core/src/datasource/mod.rs` I think this PR looks ready to go.
> 
> Thank you for your work on it

These changes introduced during Struct renaming by IDE, I missed these changes. Thanks for pointing it out. I have reverted these changes